configuration/intel_owl_nginx_http

```diff
@@ -9,6 +9,8 @@ server {
 
     server_name intel_owl.com;
 
+    server_tokens off;
```

Comment:
why this change ?

Reply:
this was a commit of mine in `develop` to remove the Nginx version banner